tasks/main.yml

```diff
@@ -10,51 +10,32 @@
   stat: path=/usr/bin/gitlab-ctl
   register: gitlab_file
 
-# Install GitLab and its dependencies (RedHat).
-- name: Install GitLab dependencies (RedHat).
-  yum: "name={{ item }} state=installed"
+# Install GitLab and its dependencies.
+- name: Install GitLab dependencies.
```

Comment:
Ah, nevermind... I was reading the wrong thing!

This is fine, ignore my earlier comment :)